.travis.yml

```diff
@@ -77,7 +77,7 @@ env:
     - PATH="$PATH:$TRAVIS_BUILD_DIR/node_modules/.bin"
     - BEARS_ZIP_URL=https://codeload.github.com/coala/coala-bears/zip
     - GRAVIS="https://raw.githubusercontent.com/DanySK/Gravis-CI/master/"
-    - JDK="adopt@1.8.0-212"
+    - JDK="graalvm-ce-java8@21.0.0"
```

Comment:
I don't know where this comes from but you should be able to use any build of `adopt@1.8.0-xxx`. Check the travis docs to see which builds are available.

Reply:
pip install language-check need jdk8 and above to run on python 3.6 and above. Any specific reason to use adopt@1.8.0-xxx? I have taken this version from Travis docs

Reply:
We would want to use openjdk and we wouldn't want to use private sources if this is one of them.

Reply:
Yeah, so graalvm is an open-source project and uses OpenJDK runtime under the hood. I found it somewhere in the supported JVM list of Travis.ci